app/jobs/health/flag_ineligible_patients_job.rb

```diff
@@ -38,8 +38,10 @@ def perform(inquiry_id)
           patient.update!(coverage_level: Health::Patient.coverage_level_managed_value,
                           previous_aco_name: previous_aco,
                           aco_name: aco)
-          aco_id = Health::AccountableCareOrganization.active.find_by(edi_name: aco)&.id
-          patient.patient_referral.update!(accountable_care_organization_id: aco_id) if aco_id.present?
+          if aco.present?
+            aco_id = Health::AccountableCareOrganization.active.find_by(edi_name: aco)&.id
+            patient.patient_referral.update!(accountable_care_organization_id: aco_id) if aco_id.present?
```

Comment:
This is already present on line 43 (trailing if)

Reply:
They are slightly different, line 41 is intended to capture the case where the ACO is blank in the 271, and the trailing if is for the case where the ACO appears in the 271, but we have no mapping...

I was worrying about the case where there is no ACO in the 271, but there is blank EDI name in the table...

Reply:
Ah!